examples/README.md

```diff
@@ -6,3 +6,11 @@ build the valve and copy the jar in the ${catalina.base}/lib directory.
 build the webapp and copy the war in the ${catalina.base}/webapps directory.
 
 the trace output is going in the catalina.out and in the header vault.param
+
+# HOW TO TEST
+simply run mvn test
```

Comment:
Put `mvn test` on a separate line or at least put \` around it -- it will get rendered differently.